### PR TITLE
sensible-editor: add page

### PR DESCRIPTION
**Pull request by RH-sdavey**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/sensible-editor.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# sensible-editor
+
+> Open the default editor.
+
+- Open a file in the default editor:
+
+`sensible-editor %{{file}}`
+
+- Open a file in the default editor, cursor at the end of the file:
```
Comment (waldyrious) [inline pages/linux/sensible-editor.md]:
```suggestion
- Open a file in the default editor, with the cursor at the end of the file:
```

File: pages/linux/sensible-editor.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`sensible-editor + %{{file}}`
+
+- Open a file in the default editor, cursor at line 10:
```
Comment (waldyrious) [inline pages/linux/sensible-editor.md]:
```suggestion
- Open a file in the default editor, with the cursor at line 10:
```

Also, is the cursor placed at the start or the end of the line?

File: pages/linux/sensible-editor.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`sensible-editor +10 %{{file}}`
+
+- Open 3 files in vertically splitted vim windows at the same time:
+
+`sensible-editor -O3 %{{file_1}} %{{file_2}} %{{file_3}}`
```
Comment (waldyrious) [inline pages/linux/sensible-editor.md]: What in this command specified that the editor should be vim?

**Comment by waldyrious** [comment]
Hey @RH-sdavey, thanks for the contribution! Please check the comments I left inline.

File: pages/linux/sensible-editor.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# sensible-editor
+
+> Open the default editor.
+
+- Open a file in the default editor:
+
+`sensible-editor %{{file}}`
```
Comment (sbrl) [inline pages/linux/sensible-editor.md]: Is the percent sign `%` here a typo?

File: pages/linux/sensible-editor.md (hunk 2 above)
Comment (RH-sdavey) [reply pages/linux/sensible-editor.md]: suggestion committed with edit, new line reads:
- Open a file in the default editor, with the cursor at the beginning of line 10:

File: pages/linux/sensible-editor.md (hunk 3 above)
Comment (RH-sdavey) [reply pages/linux/sensible-editor.md]: changed, sorry about that

File: pages/linux/sensible-editor.md (hunk 4 above)
Comment (RH-sdavey) [reply pages/linux/sensible-editor.md]: no, my misunderstanding of the convention, changed now, thanks

File: pages/linux/sensible-editor.md (hunk 3 above)
Comment (waldyrious) [reply pages/linux/sensible-editor.md]: No problem, just making sure :)

**Pull request merged**